### PR TITLE
Add OpenManipulator environment 

### PR DESCRIPTION
**Pull request by whikwon**

### CODE REVIEW

File: scripts/envs/open_manipulator/OMReacher_v0.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+Z_MAX = 0.6
+TERM_COUNT = 10
+SUC_COUNT = 10
+
+
+ACTION_DIM = 3 # Cartesian
+OBS_DIM = (100,100,3)      # POMDP
+STATE_DIM = 24        # MDP
+
+class robotEnv():
```
Comment (Curt-Park) [inline scripts/envs/open_manipulator/OMReacher_v0.py:64]: would be better if you inherit the gazebo [env class](https://github.com/erlerobot/gym-gazebo/blob/master/gym_gazebo/envs/gazebo_env.py).
Just override the interface methods.

+ add the necessary requirement as well.

Comment (Curt-Park) [reply scripts/envs/open_manipulator/OMReacher_v0.py:64]: If you wanna implement from the initial state, use this one.
[gym.Env](https://github.com/openai/gym/blob/master/gym/core.py).

Usage: https://github.com/openai/gym/blob/master/gym/envs/atari/atari_env.py

File: scripts/envs/open_manipulator/OMReacher_v0.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        """
+        # rospy.loginfo(joints_angles)
+        self.pub_gripper_position.publish(joints_angles[0])
+        self.pub_gripper_sub_position.publish(joints_angles[1])
+        self.pub_joint1_position.publish(joints_angles[2])
+        self.pub_joint2_position.publish(joints_angles[3])
+        self.pub_joint3_position.publish(joints_angles[4])
+        self.pub_joint4_position.publish(joints_angles[5])
+
+    def step(self, action=np.array([1, 1, 1, 1, 1, 1]), step=0):#overriden function
```
Comment (MrSyee) [inline scripts/envs/open_manipulator/OMReacher_v0.py:198]: should have argument only action
[ref: gym.Env](https://github.com/openai/gym/blob/93c5ca0d3d52afdc1e21905f848f2861ad3bbf80/gym/core.py#L46)

**Comment by whikwon** [comment]
New PR will be open by @subinlab with the same issue.

**Pull request closed without merging**